### PR TITLE
Improves powertool code + Changes some cases of checking for tool typepaths to tool behaviours instead.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Replaces the previous powertools code with new powertool code.
Previously every time you swap the tool mode of the jaws of life / power drill, the old one would get deleted and it would be replaced with a new version. Obviously, this is a really stupid way of doing it, since that causes all traits, atom colours etc. to be lost. This version simply changes some of the vars that needs changing and the tool_behaviour.
Also replaces some instances where instead of checking for tool_behaviour, the typepath of the tool was checked.

## Why It's Good For The Game

Code QOL.

## Changelog
:cl:
code: Refactors power tool code.
/:cl:

### CODE REVIEW

File: code/game/objects/items/tools/powertools.dm [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+	attack_verb = list("drilled", "screwed", "jabbed")
+	throw_range = 7
+
+/obj/item/powertool/hand_drill/proc/become_screwdriver()
+	icon_state = "drill_screw"
+	tool_behaviour = TOOL_SCREWDRIVER
+
+	hitsound = 'sound/items/drill_hit.ogg'
+
+	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
```
Comment (Sarchutar) [inline code/game/objects/items/tools/powertools.dm]: I think these attack_verb 's should be swapped

Comment (PowerfulBacon) [reply code/game/objects/items/tools/powertools.dm]: Good spot

Comment (PowerfulBacon) [inline code/game/objects/items/tools/powertools.dm]:
```suggestion
/obj/item/powertool/hand_drill/proc/become_wrench()
	icon_state = "drill_bolt"
	tool_behaviour = TOOL_WRENCH

	hitsound = null

	attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked"
	throw_range = 7

/obj/item/powertool/hand_drill/proc/become_screwdriver()
	icon_state = "drill_screw"
	tool_behaviour = TOOL_SCREWDRIVER

	hitsound = 'sound/items/drill_hit.ogg'

	attack_verb = list("drilled", "screwed", "jabbed")
```

**Review by park66665 — APPROVED** [review]
Did some local testing. Powertools work as intended. Welding lockers work as intended. Nothing seems to be left out. YOLO

**Pull request merged**